h/templates/app.html

```diff
@@ -28,6 +28,14 @@
          ng-click="login()"
          ng-switch-when="null">Sign in</a>
 
+      <!-- Share this page -->
+      <span class="share-dialog-toggle" ng-hide="isStream">
```

Comment:
I'd give this its own variable. `isStream` is handled by the route controllers. It's a bare property on the scope, rather than a nested property of a scope object, and so even though it's not modified it still goes against the best practice of not shadowing scope variables.

I'd just be explicit about this. `showShareButton` or something.

In the future, we will eventually follow through on separating the stream and viewer widget templates, and maybe the share dialog will move into the latter. But for now, let's at least keep our language clear and not conflate things.